pkg/cmd/admin/policy/reconcile_sccs.go

```diff
@@ -244,8 +244,8 @@ func (o *ReconcileSCCOptions) computeUpdatedSCC(expected kapi.SecurityContextCon
 	sort.StringSlice(expected.Users).Sort()
 
 	// make a copy of the expected scc here so we can ignore metadata diffs.
-	updated := expected
 	expected.ObjectMeta = actual.ObjectMeta
+	updated := expected
```

Comment:
But we ignore all ObjectMeta when calculating difference. We must preserve metadata if we ignore it, otherwise it will change unexpectedly. Another option is to copy just the labels and annotations.